pegasus/sites.v3/code.org/public/hourofcode2022.haml

```diff
@@ -36,14 +37,14 @@ theme: responsive
   %img.right{src: "/images/hoc2022-landing-page-banner-right.svg", alt: I18n.t(:hoc2022_landing_page_banner_alt_right)}
 
 %main
-  -# Hiding the NASA banner until post-launch due to mktg reasons
-  -# %aside
+  %aside.nasa-skinny-banner
     %img{src: "/images/nasa-logo.svg", alt: "NASA logo"}
-    %h1
-      Copy from NASA goes here
-    %a{href: "#", target: "_blank", rel: "noopener noreferrer"}
-      %button
-        Learn more
+    .wrapper
+      %h1=I18n.t(:hoc2022_nasa_banner_heading)
+      %p=I18n.t(:hoc2022_nasa_banner_description)
+    -# Hiding this until we get the link from NASA the week before Dec 5
+    -# %a{href: "#", target: "_blank", rel: "noopener noreferrer"}
+      %button=I18n.t(:homepage_banner_text_link)
```

Comment:
We're hiding the button until NASA sends us the link the week before CSEdWeek. Also, this button will say "Learn more"